custom_components/tuya_local/devices/dry_pury_evo_dehumidifier.yaml

```diff
@@ -0,0 +1,160 @@
+name: Argo DRY PURY EVO WF dehumidifier
+products:
+  - id: oya6k3fqen43zyi7
+    name: Argo DRY PURY EVO WF 21
+primary_entity:
+  entity: humidifier
+  class: dehumidifier
+  dps:
+    - id: 1
+      name: switch
+      type: boolean
+    - id: 2
+      name: humidity
+      type: integer
+      range:
+        min: 30
+        max: 80
+      mapping:
+        - step: 5
+    - id: 5
+      name: mode
+      type: string
+      mapping:
+        - dps_val: dehumidify
+          value: comfort
+        - dps_val: drying
+          value: laundry
+          icon: "mdi:tshirt-crew-outline"
+          icon_priority: 3
+        - dps_val: sleep
+          value: sleep
+          icon: "mdi:power-sleep"
+          icon_priority: 3
+    - id: 6
+      name: current_humidity
+      type: integer
+secondary_entities:
+  - entity: fan
+    dps:
+      - id: 1
+        name: switch
+        type: boolean
+      - id: 5
+        name: dehumidifier_mode
+        type: string
+        hidden: true
+      - id: 4
+        name: speed
+        type: string
+        mapping:
+          - dps_val: low
+            value: 50
+            constraint: dehumidifier_mode
+            conditions:
+              - dps_val: laundry
+                invalid: true
+          - dps_val: high
+            value: 100
+            constraint: dehumidifier_mode
+            conditions:
+              - dps_val: sleep
+                invalid: true
+  - entity: sensor
```

Comment:
Don't add deprecated sensors to new configs.
You can see the humidity values and history in the humidifier entity.